pipeline/cip_crosswalk.py

```diff
@@ -34,10 +35,70 @@ def __init__(
         self._load_2010_to_2020_crosswalk()
         self._load_2000_to_2010_crosswalk()
         self._load_1990_to_2000_crosswalk()
+        self._load_1985_to_1990_crosswalk()
 
         self.year_ranges = sorted(list(self.crosswalk.keys()), key=lambda x: x[0])
         self.min_year = min(yr[0] for yr in self.year_ranges)
 
+    def _parse_1990_cip_pdf(self):
```

Comment:
It seems as though a lot of this is unnecessary now that you've reduced the PDF manually. Do we still need the code that removes chapter and column headers?

Reply:
Yeah, take a look at the reduced PDF -- it still has "chapters" in it (and a few lines here and there which don't get parsed correctly)